server/services/constants.go

```diff
@@ -43,6 +43,9 @@ var (
 	transactionEventClaimDeveloperRewards                   = core.BuiltInFunctionClaimDeveloperRewards
 	transactionEventTopicInvalidMetaTransaction             = "meta transaction is invalid"
 	transactionEventTopicInvalidMetaTransactionNotEnoughGas = "meta transaction is invalid: not enough gas"
+
+	transactionEventDataExecuteOnDestContext = "ExecuteOnDestContext"
+	transactionEventDataAsyncCall            = "AsyncCall"
```

Comment:
Validated empirically. For other kinds of `event.Data`, the balance-changing operation is already captured in a SCR, thus, for those, we discard the "transferValueOnly" events.